ITE/8/README.adoc

```diff
@@ -0,0 +1,232 @@
+
+| Status
+| Active :smile:
+
+| Type
+| Process
+
+| Created
+| 2021-09-10
+
+|===
+
+
+[[abstract]]
+== Abstract
+
+This ITE proposes creating a new repository called `in-toto-library` under `in-toto` organization. This would allow
+storing _CUE_ and _OPA_ policies. This policies enforce correctness and truth of link:https://github.com/in-toto/attestation/[Attestation]
+formats against pre-defined policies by running data format validating tools.
+
+**OPA**, provides a high-level declarative language that lets you specify policy as code and
+simple APIs to offload policy decision-making from your software.
+**CUE**, an open-source data validation language and inference engine with its roots in logic programming.
+
+This ITE defines the scheme for overall structure of repository
+and provides a description for each policy.
+
+[[specification]]
+== Specification
+
+The specification text here. Describe in depth the feature that you may want to
+integrate.
+
+[[motivation]]
+== Motivation
+
+Whereas link:https://github.com/in-toto/in-toto/[in-toto] provides end-to-end implementations using several programming languages as follows:
+
+. link:https://github.com/in-toto/in-toto-rs/[in-toto-rs]
+. link:https://github.com/in-toto/in-toto-golang/[in-toto-golang]
+. link:https://github.com/in-toto/in-toto-java/[in-toto-java]
+
+Whereas in-toto provides end-to-end implementations using several programming languages, it requires responsibility
+to maintain across different languages as you know. It _may_ take a lot of time to get specs up-to-date and requires constantly follow-up.
+
+We cherish the in-toto! Aforementioned consideration might be one of the concerns why we had better to provide sustainable policies to developers.
+
+=== Use Case 1: Policy as Code
+
+link:https://www.accurics.com/resources/glossary/policy-as-code/[Policy as Code], is a general term that often refers to writing code in a high-level declarative language which describes policies.
+The purpose of Policy as Code is to codify policy definitions in software, which allows for consistent, automated assessment of policy compliance in modern software development practices.
+
+Increased security with reusable patterns and code snippets for creating additional policies.
+in-toto-library will allow the projects to move fast yet still keep information secure. By doing so, we can trust this library.
+
+=== Use Case 2: Providing Community-Owned Library of Policies
+
+link:https://github.com/open-policy-agent/gatekeeper-library[gatekeeper-library], would be a good benchmark for this use-case.
+Creating public library would allow us to manage policies easier and evaluating policies against different attestations using policy engines. We do not
+have to download policies.
+
+We can create a _source-of-truth_ by structuring information models and associated data schema such that every data element is mastered in only one place.
+
+=== Use Case 3: Verifying Attestations (OCI Registry)
```

Comment:
Let's generalize the text in this section and then point to the examples in cosign etc.